use-cases/README.md

```diff
@@ -0,0 +1,90 @@
+# Dapr use cases
+
+This document briefly describes scenarios that Dapr enables for your microservices. Each use case is accompanied with a link to learn more details.
+
+## Service invocation
+
+Dapr provides a uniform API that your microservice can use to find and reliably communicate with other microservices in your system. By using this API, your code can seamlessly take advantage of several built-in features:
+
+- HTTP and [gRPC](https://grpc.io) support
+- Robust retry logic
```

Comment:
I don't think we have this already implemented for service invocation.
/cc @yaron2